client/blocks/plan-purchase-features/index.jsx

```diff
@@ -29,47 +29,45 @@ import MonetizeSite from './monetize-site';
 import CustomDomain from './custom-domain';
 import GoogleAnalyticsStats from './google-analytics-stats';
 import HappinessSupport from './happiness-support';
-import CurrentPlanHeader from './current-plan-header';
 import JetpackAntiSpam from './jetpack-anti-spam';
 import JetpackBackupSecurity from './jetpack-backup-security';
 import JetpackReturnToDashboard from './jetpack-return-to-dashboard';
 import JetpackSurveysPolls from './jetpack-surveys-polls';
 import JetpackWordPressCom from './jetpack-wordpress-com';
-import { getPlansBySite } from 'state/sites/plans/selectors';
-import { getSelectedSite } from 'state/ui/selectors';
+import CurrentPlanHeader from './current-plan-header';
 import {
 	isPremium as isWpcomPremium,
 	isBusiness as isWpcomBusiness
 } from 'lib/products-values';
 import { isWordadsInstantActivationEligible } from 'lib/ads/utils';
+import { plansList } from 'lib/plans/constants';
 
 class PlanPurchaseFeatures extends Component {
 	static propTypes = {
 		plan: PropTypes
 			.oneOf( [ PLAN_FREE, PLAN_PERSONAL, PLAN_PREMIUM, PLAN_BUSINESS ] )
 			.isRequired,
-		selectedSite: PropTypes.object
+		selectedSite: PropTypes.object,
+		sitePlans: PropTypes.object,
+		page: PropTypes
```

Comment:
It's a bit odd to have a prop that specifies which page it's on. What does this component do differently on each page? Can we translate that into more descriptive property names?